dask/tests/test_graph_manipulation.py

```diff
@@ -237,16 +238,34 @@ def test_clone(layers):
     ],
 )
 def test_blockwise_clone_with_literals(literal):
+    """https://github.com/dask/dask/issues/8978
+
+    clone() on the result of a dask.array.blockwise operation with a (iterable) literal
+    argument
+    """
     arr = da.ones(10, chunks=1)
+    blk = da.blockwise(lambda a, l: a, "i", arr, "i", literal, None)
```

Comment:
Minor nit: I preferred the clarity of the `noop` def that was here before.

Reply:
reverted